src/test/ui/issues/issue-14227.rs

```diff
@@ -11,9 +11,9 @@
 #![allow(safe_extern_statics, warnings)]
 
 extern {
-    pub static symbol: ();
+    pub static symbol: u32;
 }
-static CRASH: () = symbol;
+static CRASH: u32 = symbol;
```

Comment:
I had to change this test because "reading" a `()` does not actually read anything...